ocs_ci/ocs/must_gather/must_gather.py

```diff
@@ -198,6 +233,17 @@ def validate_must_gather(self):
         self.print_invalid_files()
         self.compare_running_pods()
 
+    def validate_must_gather_restart(self):
```

Comment:
Is this function for validation after node restart?
If yes, please change it to validate_must_gather_after_node_restart